docs/simplesamlphp-metadata-pdostoragehandler.txt

```diff
@@ -0,0 +1,79 @@
+PDO Metadata Storage Handler
+=============================
+
+<!-- 
+	This file is written in Markdown syntax. 
+	For more information about how to use the Markdown syntax, read here:
+	http://daringfireball.net/projects/markdown/syntax
+-->
+
+
+<!-- {{TOC}} -->
+
+Introduction
+------------
+
+If you want to run a clustered simpleSAMLphp IdP service and you would like to have centralized storage for metadata, you can use the PDO metadata storage handler.
```

Comment:
We're also trying to change this. In the old days we had a mixture with the caps in the name, but now we try to use always the capitalized version: **SimpleSAMLphp**.